README.md

```diff
@@ -19,7 +19,7 @@ All code in this repository is Python 3
 # Doing a local build
 
 ## Dependencies
-  1. Linux distribution:
+  1. You will need to use a Linux distribution, as:
```

Comment:
This makes this have a really different form than everything else in the list (all of which just are the name of the needed thing).